lib/Onyx.ts

```diff
@@ -99,6 +99,8 @@ function connect<TKey extends OnyxKey>(connectOptions: ConnectOptions<TKey>): nu
     callbackToStateMapping[connectionID] = mapping as Mapping<OnyxKey>;
     callbackToStateMapping[connectionID].connectionID = connectionID;
 
+    OnyxUtils.storeKeyByConnections(mapping.key, callbackToStateMapping[connectionID].connectionID);
```

Comment:
In onyx, its best practice to be more verbose with comments than less. Could you please add a comment explaining here why we are doing this? Thanks!